app/recordtransfer/management/commands/set_domain.py

```diff
@@ -0,0 +1,61 @@
+from argparse import ArgumentParser
+
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Set the domain of the current site."""
+
+    INVALID_PREFIXES = ("http:", "https:")
+
+    help = "Sets the domain of the current site"
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        """Add arguments to the command."""
+        parser.add_argument(
+            "domain",
+            type=str,
+            help='The domain to set for the current site (e.g., "https://my.domain.com")',
```

Comment:
Sorry, one last thing:

```suggestion
            help='The domain to set for the current site (e.g., "my.domain.com")',
```

Reply:
No worries, I should've caught that.

Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/404/commits/ec21344cbc21078453b1637bc6fa7c3b379b3475